src/core/default/mappers/aws_client.py

```diff
@@ -2,10 +2,20 @@
 from typing import Callable, List, Optional, Any
 import boto3
 
-AVAILABLE_SERVICES = {"lambda", "s3", "dynamodb", "iam", "apigatewayv2", "sqs", "apigateway"}
+AVAILABLE_SERVICES = {
```

Comment:
Since we are moving the new function out of this file, can you revert this file? It looks like the only other changes are linting changes, but I would rather keep the PR's separate .